core/toolbox/separator.js

```diff
@@ -14,26 +14,25 @@
 goog.module('Blockly.ToolboxSeparator');
 goog.module.declareLegacyNamespace();
 
-goog.require('Blockly.Css');
-goog.require('Blockly.IToolboxItem');
-goog.require('Blockly.registry');
-goog.require('Blockly.ToolboxItem');
-goog.require('Blockly.utils.dom');
-goog.require('Blockly.utils.object');
-
-goog.requireType('Blockly.IToolbox');
-goog.requireType('Blockly.utils.toolbox');
+const Css = goog.require('Blockly.Css');
+const IToolbox = goog.requireType('Blockly.IToolbox');
+const IToolboxItem = goog.require('Blockly.IToolboxItem');
+const ToolboxItem = goog.require('Blockly.ToolboxItem');
+const dom = goog.require('Blockly.utils.dom');
+const object = goog.require('Blockly.utils.object');
+const registry = goog.require('Blockly.registry');
+const toolbox = goog.requireType('Blockly.utils.toolbox');
 
 
 /**
  * Class for a toolbox separator. This is the thin visual line that appears on
  * the toolbox. This item is not interactable.
- * @param {!Blockly.utils.toolbox.SeparatorInfo} separatorDef The information
+ * @param {!toolbox.SeparatorInfo} separatorDef The information
  *     needed to create a separator.
- * @param {!Blockly.IToolbox} toolbox The parent toolbox for the separator.
+ * @param {!IToolbox} toolbox The parent toolbox for the separator.
  * @constructor
- * @extends {Blockly.ToolboxItem}
- * @implements {Blockly.IToolboxItem}
+ * @extends {ToolboxItem}
+ * @implements {IToolboxItem}
```

Comment:
Is this `@implements` necessary if we're already extending a class that (I assume) implements the same interface? Might not be a good thing to fix now, but I'm curious.

Reply:
You're right. I think this implements isn't necessary because it already extends a class the implements the same interface.
I will hold on updating this PR though as it has a separate issue caused by it being imported destructured in other files.